test/lit.cfg

```
@@ -616,15 +616,33 @@ config.substitutions.append(('%build-clang-importer-objc-overlays',
 # FIXME: BEGIN -enable-source-import hackaround
 config.substitutions.append(('%clang-importer-sdk-concurrency-without-job-path',
                              '%r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
-
 config.substitutions.append(('%clang-importer-sdk-concurrency-without-job-nosource',
                              '-sdk %r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
 # FIXME: END -enable-source-import hackaround
-
 config.substitutions.append(('%clang-importer-sdk-concurrency-without-job',
                              '-enable-source-import -sdk %r -I %r ' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'),
                                                                        make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk', 'swift-modules-concurrency-without-job'))))
 
+# FIXME: BEGIN -enable-source-import hackaround
+config.substitutions.append(('%clang-importer-sdk-concurrency-owned-job-param-path',
+                             '%r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
+config.substitutions.append(('%clang-importer-sdk-concurrency-owned-job-param-nosource',
+                             '-sdk %r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
+# FIXME: END -enable-source-import hackaround
+config.substitutions.append(('%clang-importer-sdk-concurrency-owned-job-param',
+                             '-enable-source-import -sdk %r -I %r ' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'),
+                                                                       make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk', 'swift-modules-concurrency-owned-job-param'))))
+
+# FIXME: BEGIN -enable-source-import hackaround
+config.substitutions.append(('%clang-importer-sdk-concurrency-consuming-job-param-path',
+                             '%r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
+config.substitutions.append(('%clang-importer-sdk-concurrency-owned-job-param-nosource',
```

Maybe this `owned` here is a typo and you meant `consuming`? I'm not too familiar with this hackaround but my count of the owned and consuming's was uneven.

```suggestion
config.substitutions.append(('%clang-importer-sdk-concurrency-consuming-job-param-nosource',
```

---

Oh that's wrong then. I wonder if the hackaround isn't necessary anymore then 🤔